waterloop-site/src/components/SustainableTech/Footer/STEFooter.tsx

```diff
@@ -0,0 +1,213 @@
+  href: 'mailto:contact@waterloop.ca',
+})`
+  font-size: 14px;
+  margin: 0 15px;
+  color: ${fill};
+  text-decoration: none;
+  margin-top: 1.5px;
+`;
+
+const Divider = styled.hr`
+  width: 100%;
+  border-color: ${fill};
+  margin: 10px 0;
+  border-width: 1px;
+`;
+
+const PageTagsAndSocialsWrapper = styled.div`
+  height: 70px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  @media screen and (max-width: 990px) {
+    flex-direction: column;
+  }
+`;
+
+const PageTagsWrapper = styled.div`
+  display: flex;
+  flex-direction: row;
+`;
+
+const Tag = styled.p`
+  font-size: 16px;
+  color: ${fill};
+  margin-right: 50px;
+  cursor: pointer;
+`;
+
+const Tags = [
+  { text: 'Home', scrollTo: 'home-scroll' },
+  { text: 'About', scrollTo: 'about-scroll' },
+  { text: 'Schedule', scrollTo: 'schedule-scroll' },
+  { text: 'Teams', scrollTo: 'teams-scroll' },
+  { text: 'FAQ', scrollTo: 'faq-scroll' },
+];
+
+const Socials = [
+  { SVG: TwitterSVG, link: 'https://twitter.com/team_waterloop' },
+  { SVG: InstagramSVG, link: 'https://www.instagram.com/team_waterloop/' },
+  { SVG: FacebookSVG, link: 'https://www.facebook.com/teamwaterloop' },
+  { SVG: LinkedInSVG, link: 'https://www.linkedin.com/company/wloop' },
+];
+
+const SocialsWrapper = styled.div`
+  display: flex;
+`;
+
+const SocialBubble = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 50px;
+  height: 50px;
+  background-color: #fff;
+  border-radius: 50%;
+  margin-left: 25px;
+`;
+
+interface SocialProps {
+  SVG: string;
+}
+
+const SocialImg = styled.img.attrs((props: SocialProps) => {
+  return {
+    src: `${props.SVG}`,
+  };
+})<SocialProps>`
+  height: 14px;
+  width: auto;
+`;
+
+const FooterImg = styled.img.attrs({
+  src: STEFooterImg,
+})`
+  width: 100%;
+  height: 100%;
+  margin-bottom: -0.5rem;
+  background-color: #d1e6f5;
+`;
+
+const STEFooter: React.FC = () => (
+  <>
+    <FooterImg />
+    <FooterBackground>
+      <LogoAndContactWrapper>
+        <Logo />
+        <ContactDiv>
+          <ContactUs>Contact Us</ContactUs>
+          <ContactInfo>
+            <EmailIcon />{' '}
+            <ContactEmail target="_blank" rel="noopener noreferrer">
+              contact@waterloop.ca
+            </ContactEmail>
+          </ContactInfo>
+        </ContactDiv>
+      </LogoAndContactWrapper>
+      <Divider />
+      <PageTagsAndSocialsWrapper>
+        <PageTagsWrapper>
+          {Tags.map((tag) => (
+            <Tag
+              key={`ste-footer-${tag.text}-link`}
+              onClick={(): void => scrollTo(tag.scrollTo)}
+            >
+              {tag.text}
+            </Tag>
+          ))}
+        </PageTagsWrapper>
+        <SocialsWrapper>
+          {Socials.map((social, idx) => {
+            return (
+              <SocialBubble key={`ste-footer-social-link-${idx}`}>
+                <a href={social.link} target="_blank" rel="noopener noreferrer">
```

Comment:
For the social media icons in the footer, only the logo is clickable. I think it might be more intuitive to apply the link to the entire circle?

<img width="67" alt="Screen Shot 2022-03-08 at 10 28 18 AM" src="https://user-images.githubusercontent.com/68434174/157269703-c5073f5c-3a4e-445c-b9a3-4b9113bb0c3f.png">

Reply:
I agree, I've made a change that updates this and will push later.